tests/view/view/view.js

```diff
@@ -396,6 +396,35 @@ describe( 'view', () => {
 		} );
 	} );
 
+	describe( 'editing controller integration', () => {
+		it( 'should call post-fixers once if rendering model changes (_disableRendering integration)', () => {
+			const postFixerSpy = sinon.spy( () => false );
+			const changeSpy = sinon.spy();
+			const eventSpy = sinon.spy();
+
+			viewDocument.registerPostFixer( postFixerSpy );
+
+			view.on( 'render', eventSpy );
+
+			// This is set in editing controller on `model#_beforeChanges` event
+			view._renderingDisabled = true;
```

Comment:
Can you use `model.change` here instead of `_renderingDisabled`? (and most probably move this test to editing controller then).